backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/CallableReferenceLowering.kt

```diff
@@ -288,7 +308,9 @@ internal class CallableReferenceLowering(val context: Context): FileLoweringPass
                     superFunction.name,
                     Visibilities.PRIVATE,
                     Modality.FINAL,
-                    referencedFunction.returnType,
+                    if (referencedFunction.isSuspend && !superFunction.isSuspend)
```

Comment:
When is it true?

Reply:
Hmm, looks like never. Might be an artefact of some previous experiments.